### PR TITLE
DRIFT-510: Make package paths by using time from InfluxDB

### PR DESCRIPTION
**Pull request by atimin**
**Closes** DRIFT-510

### Please check if the PR fulfills these requirements

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] CHANGELOG.md have been updated (for bug fixes / features / docs)

### What kind of change does this PR introduce?

Change

### What is the current behavior?

Currently, we get paths of objects in Minio from InfluxDb. We have to store the paths as string in `src` field that takes disk space.

### What is the new behavior?

This path uses the name convention and the timestamps of records in Influxdb. It is not necessary to store the paths anymore. 

### Does this PR introduce a breaking change?** 

Nope

### CODE REVIEW

File: pkg/drift_client/drift_client.py [hunk 1]
```diff
@@ -96,18 +96,19 @@ def get_list(self, topics: List[str], timeframe: List[str]) -> Dict[str, List[st
             >>> # => {"topic-1": ['topic-1/1644750600291.dp',
             >>> #                  'topic-1/1644750601291.dp', ...] ... }
         """
-        to_request = self.__influx_client.query_data(topics, timeframe[0], timeframe[1])
-
         data = {}
+        for topic in topics:
+            to_request = self.__influx_client.query_data(
```
Comment (aschenbecherwespe) [inline pkg/drift_client/drift_client.py]: what does "to_request" mean?

**Review by aschenbecherwespe — CHANGES_REQUESTED** [review]
just don't understand one variable

File: pkg/drift_client/drift_client.py (hunk 1 above)
Comment (atimin) [reply pkg/drift_client/drift_client.py]: a good question. I think it should have name `values`

Comment (atimin) [reply pkg/drift_client/drift_client.py]: @aschenbecherwespe renamed

**Review by aschenbecherwespe — APPROVED** [review]

**Pull request merged**